.gitignore

```diff
@@ -58,7 +79,7 @@ typings/
 
 # gatsby files
```

Comment:
Just delete this please we're not using gatsby

Reply:
There was discussion at some point to use it but most volunteers are not currently familiar. We may still move to this in the future.